doc/new_boilerplate_migration.md

```diff
@@ -727,19 +727,19 @@ import 'package:over_react/over_react.dart';
 
 part 'foo.over_react.g.dart';
 
-UiFactory<FooProps> Foo = uiFunctionComponent(
+UiFactory<FooProps> Foo = uiFunction(
```

Comment:
The [RFD](https://sandbox.wdesk.com/a/QWNjb3VudB82NzE5NTMwMjcyOTQ4MjI0/doc/e601ef61809a432eb36f1b358c20a06d/r/-1/v/1/sec/e601ef61809a432eb36f1b358c20a06d_196) used both names for this function. `uiFunction` was what the ticket said to name it, but if anyone thinks it should be `uiFunctionComponent` I can update it.